agent/command/git.go

```diff
@@ -454,6 +465,10 @@ func (c *gitFetchProject) Execute(ctx context.Context, comm client.Communicator,
 		func() (bool, error) {
 			err := c.fetch(ctx, comm, logger, conf, opts)
 			if err != nil {
+				if opts.cloneParams != "" && strings.Contains(err.Error(), "exit status 129") {
+					logger.Execution().Error("Error cloning with clone params, retrying without")
```

Comment:
Do we do something similar here for any other params? I would think we wouldn't want to do this since it'd falsely make it look like some clone params were successful.

Reply:
removed